### PR TITLE
Thank you page

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description

[success/thank you page after successful registration](https://trello.com/c/ESZzSCEo/241-241-success-thank-you-page-after-successful-registration)

Thank you after user has successfully submitted the form.
image
image

## Acceptance Criteria

Fully translated and accessible

## Test Instructions

1. Fill the sign-up form correctly and submit, it should go to thank you page
2. Fill the sign-up form with mistakes and submit, it should not go to thank you page
3. Check the thank-you page on different screen sizes

## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)
- [x] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: pages/thankyou.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              src="/circle-check.svg"
+              alt="Circle with checkmark"
+            />
+          </span>
+          <div className="lg:w-2/3">
+            <p className="mb-4 text-sm lg:text-p leading-30px">
+              {t("confirmationP1")}&nbsp;
+              <a
+                className="underline hover:text-canada-footer-hover-font-blue text-canada-footer-font"
+                href="mailto: Experience@servicecanada.gc.ca"
```
Comment (var-kyle) [inline pages/thankyou.js:42]: I'm not sure if the email will ever change, but parameterising it might be a good idea.

**Review by var-kyle — COMMENTED** [review]
I just noticed a couple of things, but this is looking great! You're becoming a frontend pro!

File: pages/thankyou.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          <span className="w-full flex justify-center lg:w-1/3">
+            <img
+              className="w-80px mb-10 lg:mb-0 lg:ml-24 lg:w-160px"
+              src="/circle-check.svg"
+              alt="Circle with checkmark"
+            />
+          </span>
+          <div className="lg:w-2/3">
+            <p className="mb-4 text-sm lg:text-p leading-30px">
+              {t("confirmationP1")}&nbsp;
```
[the remark this answers is not in the document]
Comment (EphraimAssi) [reply pages/thankyou.js:39]: The &nbsp; was actually on purpose to fix a spacing issue I had.

File: pages/thankyou.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+              <a
+                className="underline hover:text-canada-footer-hover-font-blue text-canada-footer-font"
+                href={`mailto: ${t("confirmationEmail")}`}
```
Comment (var-kyle) [inline pages/thankyou.js:42]: I meant adding the email to props or as an env.

**Review by var-kyle — CHANGES_REQUESTED** [review]

**Comment by EphraimAssi** [comment]
I saw that the report a problem has the same email link in the env file, so I put that one.

**Review by will0684 — DISMISSED** [review]
Looks and works great, nailed it! 👍

File: pages/thankyou.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+              src="/circle-check.svg"
+              alt="Circle with checkmark"
+            />
+          </span>
+          <div className="lg:w-2/3">
+            <p className="mb-4 text-sm lg:text-p leading-30px">
+              {t("confirmationP1")}&nbsp;
+              <a
+                className="underline hover:text-canada-footer-hover-font-blue text-canada-footer-font"
+                href={`mailto: ${process.env.NOTIFY_REPORT_A_PROBLEM_EMAIL}`}
```
Comment (var-kyle) [inline pages/thankyou.js:42]: That's the right idea, but Nextjs needs you to add `NEXT_PUBLIC_` in front of an env to make it available to the client.

It currently will render as `mailto: undefined`
image

This change would also have to be coordinated with Marcus so he can change the env in the pipeline as well.

**Review by var-kyle — CHANGES_REQUESTED** [review]

**Review by var-kyle — APPROVED** [review]
Awesome, looks great!

Make sure Marcus updates the env in the pipeline before merging this PR though. It won't make the build fail, but we would have broken link for the email.

**Pull request merged**